dawn/js/components/Editor.js

```diff
@@ -178,6 +180,14 @@ export default React.createClass({
   stopRobot() {
     Ansible.sendMessage('stop', {});
   },
+  gameEnable() {
+    this.state.gameEnable = !this.state.gameEnable;
```

Comment:
you should use `this.setState()` to change the state rather than modifying it directly